openapi-cli/src/test/resources/expected_gen/ballerina_project/service_type.bal

```diff
@@ -1,8 +0,0 @@
-import ballerina/http;
```

Comment:
is there a reason for removing this `service_type.bal` ?

Reply:
It has been added mistakenly since the `ballerina-project` directory we only use to write the generated files and to then check the diagnostic errors. Since `service_type.bal` is also a generated file it should not be there.